src/awe/rmdparchive.cpp

```diff
@@ -169,10 +169,7 @@ std::string RMDPArchive::getResourcePath(size_t index) const {
 			path = folderEntry.name + "/" + path;
 	}
 
-	// Remove "d:/data/" prefix if present
-	if (path.find("d:/data/") == 0) path.replace(0, 8, "");
-
-	return path;
+	return Common::replace(path, "d:/data", "");
```

Comment:
I have tested  and it seems this line causes problems when loading archives in episode.cpp in line 49-53. Changing this to "d:/data/" would fix this. I do not know why this is happening, but I would like to stick with the convention of paths not starting with a /.